modules/charon-core/src/main/java/org/wso2/charon3/core/objects/EnterpriseUser.java

```diff
@@ -0,0 +1,132 @@
+package org.wso2.charon3.core.objects;
+
+import org.wso2.charon3.core.objects.plainobjects.MultiValuedComplexType;
+import org.wso2.charon3.core.schema.SCIMAttributeSchema;
+import org.wso2.charon3.core.schema.SCIMResourceTypeExtensionSchema;
+import org.wso2.charon3.core.schema.SCIMSchemaDefinitions;
+
+/**
+ * author Pascal Knueppel <br>
+ * created at: 18.03.2019 - 16:15 <br>
+ * <br>
+ * this class serves as a wrapper class for user that helps reading the enterprise user attributes
+ */
+public class EnterpriseUser extends ScimAttributeAware {
+
+    /**
+     * a user object that might hold an enterprise extension
```

Comment:
Please follow comment convention in all places, Start with caps and end with a full stop.